tools/dockerfile/test/cxx_squeeze_x64/Dockerfile

```diff
@@ -69,6 +69,9 @@ RUN apt-get update && apt-get -y install libgtest-dev libc++-dev clang && apt-ge
 RUN apt-get update && apt-get -y install python-pip && apt-get clean
 RUN pip install argparse
 
+RUN wget http://openssl.org/source/openssl-1.0.2f.tar.gz
+ADD post-git-setup.sh /
```

Comment:
With our setup, ADD is actually a bit dangerous. We currently name the docker image after building with a sha1 of the Dockerfile itself. So if you happen to update the post-git-setup.sh script later, that change will get picked up (I think), but concurrent runs of older and newer versions of our code will keep rewriting the dockerimage to use (and you'll never be sure which version you are actually running).